frontend/assets/style/_transitions.scss

```diff
@@ -1,5 +1,14 @@
 @import "_variables";
 
+@mixin reduce-motion {
+  *:not(.force-motion),
+  *:not(.force-motion)::after,
+  *:not(.force-motion)::before {
```

Comment:
I noticed some animations happen in the `::after`, `::before` pseudo elements. eg) checkbox, toggle-button.
So took that into account here too.